zebrad/tests/common/lightwalletd/send_transaction_test.rs

```diff
@@ -252,136 +245,36 @@ pub async fn run() -> Result<()> {
     Ok(())
 }
 
-/// Loads transactions from a block that's after the chain tip of the cached state.
-///
-/// We copy the cached state to avoid modifying `zebrad_state_path`.
-/// This copy is used to launch a `zebrad` instance connected to the network,
-/// which finishes synchronizing the chain.
-/// Then we load transactions from this updated state.
-///
-/// Returns a list of valid transactions that are not in any of the blocks present in the
-/// original `zebrad_state_path`.
-#[tracing::instrument]
-async fn load_transactions_from_future_blocks(
-    network: Network,
-    zebrad_state_path: PathBuf,
-) -> Result<Vec<Arc<Transaction>>> {
-    let partial_sync_height =
-        load_tip_height_from_state_directory(network, zebrad_state_path.as_ref()).await?;
-
-    tracing::info!(
-        ?partial_sync_height,
-        partial_sync_path = ?zebrad_state_path,
-        "performing full sync...",
-    );
-
-    let full_sync_path =
-        copy_state_and_perform_full_sync(network, zebrad_state_path.as_ref()).await?;
-
-    tracing::info!(?full_sync_path, "loading transactions...");
-
-    let transactions =
-        load_transactions_from_block_after(partial_sync_height, network, full_sync_path.as_ref())
-            .await?;
-
-    Ok(transactions)
-}
-
-/// Loads transactions from a block that's after the specified `height`.
+/// Loads transactions from 1-3 block(s) after the chain tip of the cached state.
```

Comment:
Nitpick: this depends on the exact value of a constant, so we could link to it, or just be vague.

```suggestion
/// Loads transactions from a few block(s) after the chain tip of the cached state.
```